### DISCUSSION TITLE
Expected formatting should be shown to the user

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5671] [Feature Requests]
<img width="531" alt="image" src="https://user-images.githubusercontent.com/37256050/206736925-a4183b29-8435-49b3-aa16-cce66bb6ff24.png">

Sort of an extension of https://github.com/specify/specify7/issues/2137 but can be solved independently so I have opened this issue separetly. The current situation leads to the data entry person not understanding the expected formatting, especially in cases where there is not a default value for the required field.

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk:
> If you remove the value from the field, the placeholder would show the expected format - not ideal, but that's what we have until this is fixed properly

@grantfitzsimmons:
> https://user-images.githubusercontent.com/37256050/206740349-47c66ced-fd2d-42f2-b6aa-c5a7d4669f08.mov
> 
> Sadly this gives me no indication at all
> 
> https://dukebryo-testability.test.specifysystems.org/specify/view/accession/new/
> 
> Same on `edge` and `testability` so nothing new

**Comment by maxpatiiuk** [comment]
See https://github.com/specify/specify7/issues/2137#issuecomment-1347057202

> For future reference: I think this happens only for UiFormatters that don't have autonumbering enabled. For example, here is the AccessionNumber UI Formatter definition:
> 
> ```
> <format system="true" name="AccessionStringFormatter" class="edu.ku.brc.specify.datamodel.Accession" fieldname="accessionNumber" default="true">
>     <field type="alphanumeric" size="10" value="AAAAAAAAAA"/>
> </format>
> ```

**Comment by CarolineDenis** [comment]
@maxpatiiuk could we discuss this issue?

**Comment by maxpatiiuk** [comment]
Sure.
In short, if you look at the first screenshot Grant posted, the error message says the value doesn't match the format, but doesn't say what the format is. There is no placeholder value in the field either (the placeholder should have been the format value).
Seems to happen to formatters without autonumbering

This should probably be fixed together with https://github.com/specify/specify7/issues/2137
And alongside adding the UI Formtter editor
